feature/cloudfront/sign/sign_cookie.go

```diff
@@ -24,6 +24,7 @@ type CookieOptions struct {
 	Domain   string
 	Secure   bool
 	SameSite http.SameSite
+	Expire   *AWSEpochTime
```

Comment:
Let's just use `time.Time` - the wrapper is for serialization which isn't necessary if we're just passing it to an `http.Cookie`.

This looks fine otherwise.

Reply:
Updated with `time.Time`.
Thanks for your feedback.